php/libraries/User.class.inc

```diff
@@ -260,12 +262,12 @@ class User extends UserPermissions
      *
      * @return string
      */
-    function getSiteNames()
+    function getSiteNames(): string
```

Comment:
getSiteName*s* is plural, shouldn't this be an array of strings, not a string?

Reply:
#4118 Addresses this change and fixes the calling code.